hasktorch/test/Torch/Typed/FunctionalSpec.hs

```diff
@@ -644,6 +644,56 @@ instance
         t' = all' @dim @keepOrDropDim t
     checkDynamicTensorAttributes t'
 
+data LstmCellSpec = LstmCellSpec
+
+instance
+  ( TensorOptions '[4 * hiddenSize, inputSize] dtype device
+  , TensorOptions '[4 * hiddenSize, hiddenSize] dtype device
+  , TensorOptions '[4 * hiddenSize] dtype device
+  , TensorOptions '[4 * hiddenSize] dtype device
+  , ( TensorOptions '[batchSize, hiddenSize] dtype device
+    , TensorOptions '[batchSize, hiddenSize] dtype device
+    )
+  , TensorOptions '[batchSize, inputSize] dtype device
+  , ( TensorOptions '[batchSize, hiddenSize] dtype device
+    , TensorOptions '[batchSize, hiddenSize] dtype device
```

Comment:
@SurajK7 
I think these duplicated constraint is not needed.
Do you have any reason?

Reply:
Yeah, you're right, sorry about that. Will fix it.